stereo_points/stereo_points.py

```diff
@@ -219,8 +271,22 @@ def onControlPointSelectedChanged(self, newNode):
             newCoordTable.AddNodeReferenceID("stereotaxia_trajLine", newNode.GetID())
             newNode.AddNodeReferenceID("stereotaxia_coordTable", newCoordTable.GetID())
             # add an observer for both nodes, whenever one is renamed, the other one is renamed as well
-            self.lObs_tableNode = newCoordTable.AddObserver(vtk.vtkCommand.ModifiedEvent, self.onCoordTableModified)
-            newNode.AddObserver(vtk.vtkCommand.ModifiedEvent, self.onControlPointNodeModified)
+            self.observers_list.append(
+                [
+                    newCoordTable,
+                    newCoordTable.AddObserver(
+                        vtk.vtkCommand.ModifiedEvent, self.onCoordTableModified
+                    ),
+                ]
+            )
+            self.observers_list.append(
+                [
+                    newNode,
+                    newNode.AddObserver(
```

Comment:
Make sure you remove the observers (when you switch to a different line, when you exit the module, or when the module is cleaned up before application exit).